server/src/main/java/keywhiz/commands/DropDeletedSecretsCommand.java

```diff
@@ -43,7 +43,7 @@ public DropDeletedSecretsCommand() {
 
   @VisibleForTesting
   public DropDeletedSecretsCommand(SecretDAO secretDAO) {
-    super("drop-deleted-secrets", "PERMANENTLY REMOVES database records for deleted secrets");
+    this();
     this.secretDAO = secretDAO;
   }
```

Comment:
This change is unrelated to the rest of the PR, but I noticed that the constructors were replicating behavior as far as calling the super class with the same string constants. By having one constructor lean on (invoke) the other we no longer need to duplicate the string values.